### ISSUE TITLE
Keyboard users cannot take advantage of "quick configure"

### ISSUE THREAD

**Issue by Madis0** [Prospector/ModMenu#598]
Keyboard users should be able to use "quick configure" the same way mouse users can - select an item and then click "enter" or at least "left" and "enter".
